bsc-plugin/src/plugin.ts

```diff
@@ -70,9 +67,7 @@ export class RooibosPlugin implements CompilerPlugin {
     }
 
     afterProgramCreate(program: Program) {
-        if (!this.isFrameworkAdded) {
```

Comment:
`this.isFrameworkAdded` was never getting set anywhere, so the if statement was worthless.